README.md

```diff
@@ -31,7 +19,7 @@ Kyma comes with the ready-to-use code snippets that you can use to test the exte
 ## Development
 
 Develop on your remote repository forked from the original repository in Go.
-See the example that uses the [`ui-api-layer`](components/ui-api-layer) project located in the `components` directory in the `kyma` repository but applies to any Go project. This set of instructions uses the recommended [`git workflow`](https://github.com/kyma-project/community/blob/master/git-workflow.md) and the general [contribution flow](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#contribute-code-or-content). Read also the [`CONTRIBUTING.md`] document that includes the contributing rules specific for this repository.
+See the example that uses the [`ui-api-layer`](./components/ui-api-layer) project located in the `components` directory in the `kyma` repository but applies to any Go project. This set of instructions uses the recommended [`git workflow`](https://github.com/kyma-project/community/blob/master/git-workflow.md) and the general [contribution flow](https://github.com/kyma-project/community/blob/master/CONTRIBUTING.md#contribute-code-or-content). Read also the [`CONTRIBUTING.md`] document that includes the contributing rules specific for this repository.
```

Comment:
1. "See the example that uses the [`ui-api-layer`](./components/ui-api-layer) project located in the `components` directory in the `kyma` repository but applies to any Go project." -> ``See the example that uses the [`ui-api-layer`](./components/ui-api-layer) project located under the `kyma/components` directory but applies to any Go project.``
2. [`CONTRIBUTING.md`] document -> there's a link missing in here

Reply:
Thanks for pointing out the link. I can shorten the path as well. As for the use of "in" or "under" I guess it is rather a matter of preference.